src/Carousel.vue

```diff
@@ -28,7 +28,6 @@
   </section>
 </template>
 <script>
-import "babel-polyfill";
```

Comment:
You need to import these polyfills somewhere - http://babeljs.io/docs/en/babel-polyfill

Reply:
I think this was removing a second babel polyfill that was added by mistake, @kjugi correct me if I'm wrong

Reply:
It should be imported into `src/index.js` as a first thing, but isn't. Also, the old `babel-polyfill` is now `@babel/pollyfil`, so if this import exists somewhere, should be updated.

Reply:
I used `'useBuiltIns'` in webpack config so we don't need to importing `babel-polyfill` now